src/DocumentFormat.OpenXml.Framework/OpenXmlPartWriter.cs

```diff
@@ -110,7 +221,6 @@ public override void WriteStartDocument()
         public override void WriteStartDocument(bool standalone)
         {
             ThrowIfObjectDisposed();
```

Comment:
nit: I kind of like the whitespace. Is there a reason you removed it?

Reply:
This was VSCode being too aggressive about formatting. I added the blank lines back